README.md

```diff
@@ -78,125 +78,140 @@ settings](config/babel/.babelrc).
 
 ```
 $ builder help builder-react-component
-[builder:help]
 
 Usage:
 
-  builder [action] [task]
+  builder builder-react-component [flags] [task]
```

Comment:
@coopy -- Yep. Upstream PR for this at https://github.com/FormidableLabs/builder/pull/52